### PR TITLE
[MIRROR] Adds toggleable darkmode to the game!

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/43072
----------------------------
## About The Pull Request

Good morning /tg/, this PR adds dark-theme to replace the aging white design. You can use the "toggle darkmode" verb to opt-out of this change.

## Why It's Good For The Game

White theme still default.

This involves changing a few colours to match both light and dark mode, one of them being the OOC colour.

Demo:
https://gyazo.com/49c8a4637e81a654326b9a4aa5c05c38

WHY SO MUCH WINSET

If you can think of a better way please review, but I asked Lummox and he suggested winset() calls.

If you change ANY PART OF THE SKIN you will need to update the winset calls here too!

Why is darktheme on by default?
This PR is a port of my own work at:
https://github.com/DDMers/Deep-Space-13/pull/32

Where I decided to make it a default. If you object then I can change this (though it'll be a hassle)

EDIT: Thanks for the reddit gold kind stranger!

## Changelog
:cl: Kmc2000
add: Added darkmode! You can opt-in to this by clicking the settings icon on goonchat, and looking for the toggle darkmode button.
/:cl:

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
GOD HAS COME TO REAP THE SINNERS

**Comment by k-k-karma** [comment]
i like light mode; modularising out

**Comment by yoyobatty** [comment]
? You can toggle it, also I say we keep this.

**Comment by RafRoq** [comment]
Please my eyes burn playing in the dark

**Comment by RafRoq** [comment]
Don't listen to karma he is a fucker

**Comment by Carbonhell** [comment]
This may be very broken. Careful.

**Comment by Jujumatic** [comment]
modularise it to make light mode the default but keep dark mode as toggleable because most people will likely change to that.

and if we make the default as dark mode it's going to fuck with a lot of people

File: code/modules/goonchat/browserassets/js/browserOutput.js [hunk 1]
```diff
@@ -456,6 +457,19 @@ function toHex(n) {
...
 }
 
+function swap() { //Swap to darkmode
+	if (opts.darkmode){
+		document.getElementById("sheetofstyles").href = "browserOutput_white.css";
+		opts.darkmode = false;
+		runByond('?_src_=chat&proc=swaptolightmode');
+	} else {
+		document.getElementById("sheetofstyles").href = "browserOutput.css";
+		opts.darkmode = true;
```
Comment (Jujumatic) [inline code/modules/goonchat/browserassets/js/browserOutput.js:467]: ok nevermind this is probably really easy to change the default

Comment (yoyobatty) [reply code/modules/goonchat/browserassets/js/browserOutput.js:467]: Just code out the 1's and 0's bro

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**